dashboard/src/components/templates/app/WebsiteSettings.tsx

```diff
@@ -255,13 +255,16 @@ export const WebsiteSetting = (props: WebsiteSettingProps) => {
                     }}
                     {...fieldProps}
                     options={
-                      systemInfo()?.domains.map((domain) => {
-                        const domainName = domain.domain.replace(/\*/g, '')
-                        return {
-                          value: domain.domain,
-                          label: domainName,
-                        }
-                      }) ?? []
+                      // 占有されているドメインはoptionに表示しない
+                      systemInfo()
+                        ?.domains.filter((domain) => !domain.alreadyBound)
```

Comment:
このようにしないと専有しているアプリで壊れませんか？
(細かい書き方は知らない)
```suggestion
                        ?.domains.filter((domain) => !domain.alreadyBound && props.formStore.website.domain !== domain.domain)
```